server/.env

```diff
@@ -1,2 +1,5 @@
 TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Michael Singh, Dan Proctor, Michael Braga"
-MONGODB_URI=mongodb://localhost/team-pine-cone
\ No newline at end of file
+MONGODB_URI=mongodb://localhost/team-pine-cone
+ACCESS_TOKEN_NAME=JWT
+ACCESS_TOKEN_SECRET=C9ARo0VUvaB9LG9An8ax
```

Comment:
It would be best to put this into a .env file

Reply:
Is it not in a .env file?

Reply:
Apologies, I mean a non-tracked .env file. It's not good to put API keys into tracked code as anyone who can see this repo (currently anyone on the internet) can copy and use your API key.

Reply:
I think right now it's ok because these are just local credentials. But when deploy the app, we'll have sensitive info that will have to be stored somewhere else